lib/deployments.js

```diff
@@ -24,18 +24,23 @@ async function createPrediction(deployment_owner, deployment_name, options) {
     }
   }
 
+  const headers = {};
+  if (block) {
+    headers["X-Sync"] = "true";
```

Comment:
Let's use our time this afternoon to decide on the header, after using the API for a bit I'm back in the camp to use `Prefer: respond-sync`. It just feels more thought through.